apps/costcontrol/style/views/telephony.css

```diff
@@ -31,27 +31,42 @@
   border: none;
 }
 
-#reset-date,
-#telephony-tab li dd {
+#reset-date {
+  font-size: 1.4rem;
+  font-weight: 600;
+  text-align: end;
+  vertical-align: baseline;
+}
+html[dir="ltr"] #reset-date {
   float: right;
+}
+html[dir="rtl"] #reset-date {
+  float: left;
+}
+
+#telephony-tab li dd {
   font-size: 1.6rem;
   font-weight: 600;
-  text-align: right;
+  text-align: end;
+  display: block;
+  overflow: hidden;
+}
+html[dir="ltr"] #telephony-tab li dd,
```

Comment:
Happy to interate on this PR and flexbox these floats (and get rid of these overrides) but need to figure out how to see the #telephony tab